CHANGELOG.md

```diff
@@ -1,7 +1,37 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
-# [v1.7.1]
+# [v1.8.0] (Unreleased)
+### Notes
+- This release extends supports of the SDK to Oneview API Version 2400.
+- Minor Bug Fixes
```

Comment:
Since you already mentioned bugs & enhancements below, is it required to mention it here also..?

Reply:
following similar pattern in previous releases.